<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Aravind" name="my_own_created_htmlpages_Test.DashboardPageTest" tests="7" failures="1" timestamp="03 Jun 2025 14:34:57 GMT" time="74.017" errors="0">
  <testcase name="testFillNameAfterLogin" time="9.876" classname="my_own_created_htmlpages_Test.DashboardPageTest"/>
  <testcase name="testSelectGenderAfterLogin" time="8.027" classname="my_own_created_htmlpages_Test.DashboardPageTest"/>
  <testcase name="testSelectSkillAfterLogin" time="10.345" classname="my_own_created_htmlpages_Test.DashboardPageTest"/>
  <testcase name="testSelectCountryAfterLogin" time="8.479" classname="my_own_created_htmlpages_Test.DashboardPageTest"/>
  <testcase name="testUploadFileAfterLogin" time="7.970" classname="my_own_created_htmlpages_Test.DashboardPageTest"/>
  <testcase name="testClickAlertAndSaveAfterLogin" time="20.978" classname="my_own_created_htmlpages_Test.DashboardPageTest"/>
  <testcase name="testLogoutAfterLogin" time="8.342" classname="my_own_created_htmlpages_Test.DashboardPageTest">
    <failure type="java.lang.AssertionError" message="Logout failed or redirected to incorrect page expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Logout failed or redirected to incorrect page expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at my_own_created_htmlpages_Test.DashboardPageTest.testLogoutAfterLogin(DashboardPageTest.java:96)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testLogoutAfterLogin -->
</testsuite> <!-- my_own_created_htmlpages_Test.DashboardPageTest -->
